cmd/gaia/cli_test/cli_test.go

```diff
@@ -59,7 +59,7 @@ func TestGaiaCLISend(t *testing.T) {
 	fooAcc := executeGetAccount(t, fmt.Sprintf("gaiacli account %v %v", fooCech, flags))
 	require.Equal(t, int64(50), fooAcc.GetCoins().AmountOf("steak").Int64())
 
-	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --name=foo", flags, barCech), pass)
+	executeWrite(t, fmt.Sprintf("gaiacli send %v --amount=10steak --to=%v --from=foo", flags, barCech), pass)
```

Comment:
Cool so what's being done here is simply swapping `--from` for `--name` - One thing that I think we should also implement would be the ability to provide a bech address in replacement of the key-name which could also be handled by `--from`

Reply:
I agree, we should add that. Should that be added in a separate PR though?

Reply:
I don't think we need to block this PR on that - but you're motivated to do so it would be great to have here too

Reply:
Sure I can do it, I'll do it as a separate PR though